import-automation/executor/app/executor/scheduler_job_manager.py

```diff
@@ -91,10 +86,22 @@ def schedule_on_commit(github: github_api.GitHubRepoAPI,
                                                'No issues')
 
 
-def create_or_update_import_schedule(absolute_import_name: str, schedule: str,
+def create_or_update_import_schedule(absolute_import_name: str,
+                                     import_spec: dict,
                                      config: configs.ExecutorConfig,
                                      scheduler_config_dict: Dict):
     """Create/Update the import schedule for 1 import."""
+    schedule = import_spec.get('cron_schedule')
+    if not schedule:
+        raise KeyError(
+            f'cron_schedule not found in manifest for {absolute_import_name}')
+    resources = {"cpu": "2", "memory": "4G"}  # default resources.
+    if 'resource_limits' in import_spec:
+        resources = import_spec['resource_limits']
```

Comment:
can we do `resources.update(import_spec['resource_limits'])` so that manifest.json can selectively override just one of the cpu/mem resource also?